include/pfs/types.hpp

```diff
@@ -620,6 +620,16 @@ struct net_route
     unsigned int irtt;
 };
 
+struct net_arp
+{
+    std::string ipaddr;
```

Comment:
The library tries to follow a convention of underscores between words. Can we change that to `ip_address`?

Reply:
Done